llvm/lib/Target/LoongArch/LoongArchInstrInfo.td

```diff
@@ -1054,6 +1068,8 @@ def AMCAS__DB_D  : AMCAS_3R<0x385b8000>;
 def LL_D : LLBase<0x22000000>;
 def SC_D : SCBase<0x23000000>;
 def SC_Q : SCBase_128<0x38570000>;
+def LLACQ_W : LLBase_ACQ<0x38578000>;
+def SCREL_W : SCBase_REL<0x38578400>;
```

Comment:
This can be a separate change that can be fast-tracked?

Reply:
They aren't new additions; rather, they are being moved out of the `Has32S` predicates.

Reply:
Ah I see. It seems my "Find in page" function was a bit unreliable.